src/test/java/ch/uzh/soprafs22/groupmatcher/service/EmailServiceTest.java

```diff
@@ -37,6 +39,9 @@ class EmailServiceTest {
     @MockBean
     MatcherRepository matcherRepository;
 
+    @MockBean
```

Comment:
You don't need to mock the Matcher entity in general so you can just remove this, rather what we need is to create a fake matcher (which already happens on line 54 + the setup() function)